packages/app/src/resource/AuditEventPage.tsx

```diff
@@ -0,0 +1,29 @@
+import { Operator, SearchRequest } from '@medplum/core';
+import { ResourceType } from '@medplum/fhirtypes';
+import { Document, MemoizedSearchControl } from '@medplum/react';
+import React, { useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+export function AuditEventPage(): JSX.Element | null {
+  const { resourceType, id } = useParams() as { resourceType: ResourceType; id: string };
+  const navigate = useNavigate();
+  const [search, setSearch] = useState<SearchRequest>({
+    resourceType: 'AuditEvent',
+    filters: [{ code: 'entity', operator: Operator.EQUALS, value: `${resourceType}/${id}` }],
+    fields: ['id', '_lastUpdated'],
+    sortRules: [{ code: '-_lastUpdated' }],
+    count: 20,
+  });
+
+  return (
+    <Document>
+      <MemoizedSearchControl
+        search={search}
+        onClick={(e) => navigate(`/${e.resource.resourceType}/${e.resource.id}`)}
+        onChange={(e) => setSearch(e.definition)}
+        hideFilters
+        hideToolbar
```

Comment:
If it's possible, I think it would actually be helpful to have the toolbar. For instance, it would be nice to be able to filter to all `AuditEvents` where the `outcome != 0` indicating an error